OPHD/UI/PopulationPanel.cpp

```diff
@@ -126,19 +130,41 @@ void PopulationPanel::update()
 
 	position.y += fontBoldHeight + constants::Margin;
 	const auto textOffset = NAS2D::Vector{IconSize + constants::Margin, (IconSize / 2) - (fontHeight / 2)};
+	const int populationStatisticsWidth = mFont.width(constants::PopulationBreakdown) + constants::Margin;
+
 	for (const auto& [imageRect, personCount, personRole] : populationData)
 	{
 		renderer.drawSubImage(mIcons, position, imageRect);
 
-		const auto roleCount = std::to_string(personCount);
 		renderer.drawText(mFont, personRole + ": ", position + textOffset);
 
-		const NAS2D::Point<int> labelPosition = {positionX() + mPopulationPanelWidth - mFont.width(roleCount) - constants::Margin , position.y + textOffset.y};
-		renderer.drawText(mFont, roleCount, labelPosition);
+		const auto personCountString = std::to_string(personCount);
+		const NAS2D::Point<int> labelPosition = {positionX() + mPopulationPanelWidth - mFont.width(personCountString) - constants::Margin, position.y + textOffset.y};
+		renderer.drawText(mFont, personCountString, labelPosition);
+
```

Comment:
Very minor point here, though I was just thinking this refactoring is a bit independent of the new feature you're adding. That could make it a good candidate to do this change as a separate commit, before the changes to add your new feature.

No changes required here. Just noting it for future reference as a way of potentially communicating a change is a cleanup to old code versus part of implementing a new feature.